packages/eslint-config-edx/.eslintrc.json

```diff
@@ -6,14 +6,19 @@
         "jasmine": true,
         "jquery": true
     },
-    "extends": "airbnb-base",
+    "extends": "airbnb",
     "parserOptions": {
-        "ecmaVersion": 8
+        "ecmaVersion": 8,
+        "ecmaFeatures": {
+          "jsx": true
+        }
     },
     "plugins": [
         "dollar-sign"
     ],
     "rules": {
-        "dollar-sign/dollar-sign": ["error", "ignoreProperties"]
+        "dollar-sign/dollar-sign": ["error", "ignoreProperties"],
+        "react/no-array-index-key": "off",
```

Comment:
Why is this being turned off? [The docs](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md) say it could result in unnecessary re-renders.